### PR TITLE
Gnosis Safe Apps v1.5.0

### PR DESCRIPTION
**Pull request by nicosampler**
#### What's new
* WalletConnect app (https://github.com/gnosis/safe-react/issues/1239).

#### What has changed
* safe-apps-sdk bump.

#### Bugfix
* TX-Builder: Support for Array and Tuples (#23)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,29 +4,34 @@
   "private": true,
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
-    "@gnosis.pm/safe-apps-sdk": "^0.1.1",
-    "@gnosis.pm/safe-react-components": "^0.1.3",
+    "@gnosis.pm/safe-apps-sdk": "0.4.0",
+    "@gnosis.pm/safe-react-components": "https://github.com/gnosis/safe-react-components.git#be51ca5",
     "@material-ui/core": "^4.9.0",
+    "@rmeissner/safe-apps-react-sdk": "^0.3.4",
```
Comment (dasanra) [inline package.json:10]: Should functions in this repository be on some @gnosis repository? Is something pending?

**Review by dasanra — APPROVED** [review]

**Comment by FlacoJones** [comment]
Is this still effective? Is there some example code for how to format the tuple for Transaction Builder?

Screenshot 2023-02-09 at 10 50 51 AM

**Comment by mantasfam** [comment]
What format I should use for this? Any example please
`erc20Rewards (tuple(address,uint256)[])`

**Comment by AlissonRS** [comment]
For anyone struggling with this, I found the answer on #701 

For input of type `tuple(address,uint256)[]`

...you can provide an array of tuples like this:

`[["0x0000000000000000000000000000000000000000", "2"]]`

It's a bit strange because block explorers accept the format below, but Safe Wallet doesn't:

`[{ "address": "0x07A125527651e45e22c31a4ba25440fd582A21FB", "myNumber": 2 }]`

**Pull request merged**